script.js

```diff
@@ -12,4 +29,286 @@ function closePopUp(){
   pop_up_menu.style.display="none";
  }
 
-//popup window will go here for project details
\ No newline at end of file
+
+
+
+
+//workinfo object 
+const workInfo={
+  myRecentPr: 'My Recent Projects',
+  seeProject: 'See Project',
+  bgImg: './images/Img Placeholder (1).png',
+  prTitle1:'Profesional Art Printing Data More',
+  prTitle2:'Multi-Post Stories',
+  prDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required. This has  been the industry standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+  techList: ['HTML','CSS','Javascript']
+};
+
+
+// markup work section from the wworkinfo array object
+var mainElmnt=document.querySelector('main');
+const divSection=document.createElement('div');
+  mainElmnt.insertBefore(divSection,aboutMeSection);
```

Comment:
- [ ] Your work section is not displaying on your webpage. Kindly fix this please. I suggest you add an empty section element in your index.html and then call it in your js file and add contents to it using.
![image](https://user-images.githubusercontent.com/94551484/217561828-c684c2a6-2042-4065-b521-d24a36450bbc.png)